descriptors/ModuleDescriptor-template.json

```diff
@@ -33,6 +33,17 @@
         }
       ]
     },
+    {
+      "id": "item-storage-batch-sync",
+      "version": "1.0",
```

Comment:
Personally, I'd consider marking these interfaces as experimental, until streaming support / too large payloads etc are in place